docs/shader-modules/project64.md

```diff
@@ -43,16 +41,19 @@ Projected position in the clipspace.
 
 64 bit counterpart of the `project` modules `project_position`
 
-`void project_position_fp64(vec4 position_fp64, out vec2 out_val[2])`
-
-`void project_position_fp64(vec2 position, vec2 position64xyLow, out vec2 out_val[2])`
+```glsl
+void project_position_fp64(vec4 position_fp64, out vec2 out_val[2])
+void project_position_fp64(vec2 position, vec2 position64xyLow, out vec2 out_val[2])
+```
 
 
-### project_to_clipspace_fp64
+### project_common_position_to_clipspace_fp64
```

Comment:
Total Nit: As I read these, I wonder if `project_position_common_to_clipspace` would be been better, i.e putting what is being projected first and the from-to spaces last?

I think a combined audit of both JS and GLSL projection APIs would have been ideal, to ensure parity and naming consistency, but is likely too much to undertake at this stage.

Reply:
It would read a little strange to me if the `to_*` part is skipped, e.g. `project_pixel_size` vs. `project_size_pixel`.

I don't have a strong opinion either way.